### PR TITLE
getObject from S3 in readProfileHandler

### PR DESCRIPTION
**Pull request by Balou9**
Lambda execution failed with status 200 due to customer function error: Missing required key 'Bucket' in params.

### CODE REVIEW

File: lambdas/profiles/read/index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const s3 = new S3({
 
 module.exports.handler = async function handler ({
   event,
+  context,
   headers,
   pathParameters
 }) {
```
Comment (chiefbiiko) [inline lambdas/profiles/read/index.js]: still wrong handler signature;; you are destructuring the first argument to the handler
should be: `handler(event, context)`
`event` structure visible [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/37ef96abd4c38621aebe374ed67a3f38a51f55f3/types/aws-lambda/trigger/api-gateway-proxy.d.ts#L65)

**Comment by chiefbiiko** [comment]
sorry, will nohc mehr reviews geben aber github zeigt mir nicht ide buttons
egal, dann hier

so ithink the `exists` check doesnt make sense - why are you doing it?
also i would bind the `Bucket` param to the s3 instance when creating it 
and then omit it afterwards
```
var s3 = new AWS.S3({apiVersion: "...", params: { Bucket: process.env.BUCKET_NAME }})
```

**Comment by Balou9** [comment]
I was checking if the key exists due to former errors. You're right it doesn't make sense.

**Comment by Balou9** [comment]
readHandler Logs: Lambda execution failed with status 200 due to customer function error: Access Denied.

File: lambdas/profiles/read/index.js [hunk 2]
```diff
@@ -1,39 +1,27 @@
...
+    "BUCKET:::", s3,
   )
 
   const params = {
-    Bucket: process.env.BUCKET_NAME,
-    Key: pathParameters.profileName,
+    Key: event.pathParameters.profileName,
   }
 
   const payload = await s3.getObject(params).promise()
```
Comment (chiefbiiko) [inline lambdas/profiles/read/index.js]: wrap this in a `try {} catch() {}` and return a http error accordingly
and also i would delete the stack
before redeploying with this change

**Pull request merged**